src/test/cls_lock/test_cls_lock.cc

```diff
@@ -503,22 +486,22 @@ TEST(ClsLock, TestExclusiveEphemeralBasic) {
   ASSERT_EQ(0, ioctx.stat(oid1, &size, &mod_time));
   sleep(2);
   int r1 = l1.unlock(&ioctx, oid1);
```

Comment:
No need to print the obvious "unlock should return 0 return".  Just use to `ASSERT_EQ`, as in the original commit a289f2d8654cf4b865430465b87299d3618b41c8:
```suggestion
  ASSERT_EQ(0, l1.unlock(&ioctx, oid1));
```